src/domain/occurrence/EditOccurrencePage.tsx

```diff
@@ -152,6 +167,19 @@ const EditOccurrencePage: React.FC = () => {
     }
   };
 
+  const handleDeleteOccurrence = async (
+    occurrence: OccurrenceFieldsFragment
+  ) => {
+    try {
+      await deleteOccurrence({ variables: { input: { id: occurrence.id } } });
+      refetchEvent();
+    } catch (e) {
+      toast(t('occurrences.deleteError'), {
+        type: toast.TYPE.ERROR,
+      });
```

Comment:
You could also write this line as
```
toast.error(t('occurrences.deleteError'));
```
In that case could can test it easier with Jest (If you want to test the error case). e.g. `toast.error = jest.fn()`

Reply:
fixed